### DISCUSSION TITLE
Google Analytics

### DISCUSSION THREAD

**Discussion by organicnz** [AntlerVC/firetable#803] [Idea / Feature Request]
Hi folks,

Would be awesome to have a Google Analytics template :)

Thank you :)

**Comment by harinij** [comment]
@organicnz great suggestion. We can definitely look into adding one. Is there anything specific you are looking to see? If you can share a bit more about your use case it would be super useful.

**Comment by organicnz** [comment]
@harinij I would like to get general analytics from Google Analytics v3 and v4 and visualise if possible. So far this has been implemented manually in [Google Spreadsheets](https://www.loom.com/i/c5e078a9401b4630ae452401459a838e) and added screenshot in Google Spreadsheet from [Google Analytics](https://www.loom.com/i/8a06854c1ccf44199ae51b08e3b8d96d), [Yandex Metrika](https://www.loom.com/i/76aadfdcc70f47db9cfb9d26a79e0c7d), [Ammo CRM](https://www.loom.com/i/4b9a27fc4bb34ab8ab4983604ab5d6b4), [Telegram](https://www.loom.com/i/2f230286ce814c9aa38e15639adabce6) and [Instagram](https://www.loom.com/i/8a64968f866a4fd19d87dec01a06d5be). 

It would also be nice to combine this with other analytics from Telegram, Instagram, Yandex Metrika, Ammo CRM and Amplitude into one collection in the future :)

**Comment by harinij** [comment]
@organicnz some of these dashboards seem very comprehensive and might not be ideal to pull all the data points into one collection. If there are specific data points of interest to you at table level/for use in your app.

For example, we are exploring in this [demo of UTM tracking link](https://demo.rowy.io/table/utmLinks), ability to see the lifetime clicks per campaign URL via google analytics.

**Comment by organicnz** [comment]
@harinij it would be great to get some data points for each month :)
- Visitors to the site (Int)
- Dynamic (Percentage)
- Peak (Date)
- Visitors to Peak (Int)
- Geography 
- Revenues (USD)
- Revenues (Percentage)
- Sessions (Int)
- Sessions (Percentage) 
- Bounce Rate Percentage) 

Thank you. I'll have a look :)
